frontend/src/components/UI/NavBar.tsx

```
@@ -31,28 +34,25 @@ import ProfilePicture from "./user/ProfilePicture";
-    // Re-Render on window resize
-    const windowSize = useWindowSize();
-    console.log(windowSize);
     // Router states
     const navigate = useNavigate();
     const location = useLocation();
     // Conditional Variables to NavBar changes
-    const user_type: number =  User.data.type as number;
+    const user_type: UserTypeEnum =  User.data.type;
     const buttons: React.ReactElement[] | undefined[] = [
         <img src={notification_icon} alt="Notification" className="bell" style={{ display: window.innerWidth > window.innerHeight ? "none" : "block"}} />,
         <a href="#footer"><img src={about_us_icon} alt="About Us" /></a>,
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)